bign256/Cargo.toml

```
@@ -0,0 +1,45 @@
+[package]
+name = "bign256"
+version = "0.1.0"
```

This can be bumped at the time of release as well.

```suggestion
version = "0.0.0"
```